optimizely/helpers/audience.py

```diff
@@ -49,14 +65,28 @@ def evaluate_audience(audienceId):
     if audience is None:
       return None
 
-    return condition_tree_evaluator.evaluate(
+    logger.debug(logs.EVALUATING_AUDIENCE_WITH_CONDITIONS.format(audienceId, audience.conditions))
+
+    result = condition_tree_evaluator.evaluate(
       audience.conditionStructure,
       lambda index: evaluate_custom_attr(audienceId, index)
     )
 
+    result_log = str(result) if result is not None else 'UNKNOWN'
+    logger.debug(logs.AUDIENCE_EVALUATION_RESULT.format(audienceId, result_log))
```

Comment:
doc suggests that this should be info. I have made it debug to avoid logs being too verbose. 
At info level we should be interested in the overall result, which we are already logging outside on line#87